src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,83 @@
+import { useContext, FC } from 'react';
+import { DispatchContext, StateContext } from '../../store/todoReducer';
+import { Action } from '../../types/actions';
+import { deleteTodo } from '../../api/todos';
+import { filterAction } from '../../constants/filterActions';
+
+type Props = {
+  onError: (message: string) => void;
+  onCoverShow: React.Dispatch<React.SetStateAction<number[]>>;
+};
+
+export const Footer: FC<Props> = ({ onError, onCoverShow }) => {
+  const { todos, filter } = useContext(StateContext);
+  const dispatch = useContext(DispatchContext);
+
+  const isActiveTodo = todos.some(todo => todo.completed);
```

Comment:
`hasActiveTodo` would be a better choice for this constant's name. `isActiveTodo` implies to indicate if a certain todo is active - which is not the case here.